### PR TITLE
displaying corgi image number

### PR DESCRIPTION
**Pull request by baconlover2020**
Implementing enhanced described in issue #52

### CODE REVIEW

File: Cogs/StudentCommands.py [hunk 1]
```diff
@@ -28,15 +28,12 @@ async def corgme(self, ctx, number=-1):
...
         print(images[0])
-
-        # Pick a random image
-        if number != -1 and (0 < number < len(images)):
-            image = images[number]
-        else:
-            image = images[randint(0, len(images) - 1)]
-
+        #Generates a random number if no number is given
+        if number == -1:
```
Comment (tux2603) [inline Cogs/StudentCommands.py:32]: This changes the functionality of the code. In the original version, this if statement also checks that the specified number is within the range of valid indices.

I'd recommend changing this line to `if 0 <= number < len(images):`

**Review by tux2603 — CHANGES_REQUESTED** [review]

**Comment by CodingPenguin1** [comment]
@baconlover2020 Might I ask who you are? While this repo is public, we haven't announced it or anything. Are you a Wright State student?

File: Cogs/StudentCommands.py (hunk 1 above)
Comment (baconlover2020) [reply Cogs/StudentCommands.py:32]: Ohh I see, I'll fix that

**Comment by baconlover2020** [comment]
@CodingPenguin1 I am not a Wright State student :p
I was just looking up issues on github and came across this repo.
I still a beginner, so its my first time contributing to projects like this one.

**Comment by CodingPenguin1** [comment]
Huh. Interesting that GitHub made this public enough to be stumbled across

**Review by tux2603 — APPROVED** [review]
Looks good to me

**Comment by tux2603** [comment]
@CodingPenguin1 do you see any potential issues?

File: Cogs/StudentCommands.py [hunk 2]
```diff
@@ -28,15 +28,12 @@ async def corgme(self, ctx, number=-1):
...
         images = ['dogs/corgis/' + path.name for path in Path('dogs').rglob('*.*')]
         print(images[0])
-
-        # Pick a random image
-        if number != -1 and (0 < number < len(images)):
-            image = images[number]
-        else:
-            image = images[randint(0, len(images) - 1)]
-
+        #Generates a random number if no number is given
```
Comment (CodingPenguin1) [inline Cogs/StudentCommands.py:31]: Please put a new line between 30 and 31, and a space after # in 31

**Review by CodingPenguin1 — CHANGES_REQUESTED** [review]

File: Cogs/StudentCommands.py (hunk 2 above)
Comment (baconlover2020) [reply Cogs/StudentCommands.py]: alright I think I got it

**Review by CodingPenguin1 — APPROVED** [review]

**Comment by tux2603** [comment]
Closes #52

**Pull request merged**